src/main/java/com/google/ehub/servlets/UserDataManager.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+
+public class UserDataManager {
+  public static UserProfile getUserProfile(String email) {
+    Filter propertyFilter = new FilterPredicate(EMAIL_PROPERTY_KEY, FilterOperator.EQUAL, email);
```

Comment:
The use of ```EMAIL_PROPERTY_KEY``` and ```PROFILE_ITEM_KIND``` come from another class(ProfileDatastore) in the same package, but to a reader it might be confusing to figure out where this strings come from. I would suggest either adding this values as private member variables of the ```UserDataManager``` class or create a new class that holds all the common keys that deal with users.

Reply:
In the new PR you will see that I created get methods to get the EMAIL_PROPERTY_KEY and PROFILE_ITEM_KIND